tables/table.py

```diff
@@ -1689,7 +1760,10 @@ def read_sorted(self, sortby, checkCSI=False, field=None,
         coords = index[start:stop:step]
         return self.read_coordinates(coords, field)
 
-    def iterrows(self, start=None, stop=None, step=None):
+    def iterrows(self,
+                 start: Optional[int]=None,
+                 stop: Optional[int]=None,
+                 step: Optional[int]=None) -> Iterator:
```

Comment:
See the comment on `__iter__`

Reply:
As recommended changed to `Iterator[tableextension.Row]`.